src/master-playlist-controller.js

```diff
@@ -909,11 +927,17 @@ export class MasterPlaylistController extends videojs.EventTarget {
 
     // Select a new playlist
     const nextPlaylist = this.selectPlaylist();
+
+    if (!nextPlaylist) {
+      this.error = 'Playback cannot continue. No playlists are working.';
```

Comment:
Since technically the playlists can be working but not supported, maybe something like

```suggestion
      this.error = 'Playback cannot continue. No available working or supported playlists.';
```